pkg/sql/model.go

```diff
@@ -86,7 +86,6 @@ func (m *model) saveDB(db *DB, table string, session *pb.Session) (e error) {
 	if e != nil {
 		return fmt.Errorf("cannot create sqlfs file %s: %v", table, e)
 	}
-	defer sqlf.Close()
```

Comment:
How about `defer`ing with the new `Close` code in a lambda?

Reply:
I believe should check the error of `hiveWriter.Close`, I updated this PR follow https://www.joeshaw.org/dont-defer-close-on-writable-files/ that uses a safe way to close sqlfs.

Reply:
Is it safe if we call `hiveWriter.Close` twice?